src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,52 @@
 
 class Animal {
   // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
+
+  die() {
+    const index = Animal.alive.indexOf(this);
+
+    if (index > -1) {
+      Animal.alive.splice(index, 1);
+    }
```

Comment:
The `die` method uses `indexOf` and `splice` to remove the animal from the `Animal.alive` array, which is not recommended due to performance reasons. It would be better to use the `filter` method in the `updateAlive` static method to remove dead animals from the array.